### ISSUE TITLE
Check batch configurations in the steps

### ISSUE THREAD

**Issue by jsemler** [LLNL/maestrowf#238]
When writing a study step it’s easy to place the batch and depends configuration under the `run` key instead of the `cmd` key. This causes the step to not be scheduled. The syntax is easy to overlook when debugging.

Maestro will submit the step in batch for the following configuration:

```yaml
- name: say-hello
      description: Say hello to the world!
      run:
          cmd: |
            echo "Hello, World!" > hello_world.txt
          nodes: 1
          procs: 1
          walltime: "00:05:00"
          depends: []
```

Maestro will run this step locally:

```yaml
- name: say-hello
      description: Say hello to the world!
      run:
          cmd: |
            echo "Hello, World!" > hello_world.txt
      nodes: 1
      procs: 1
      walltime: "00:05:00"
      depends: []
```

It would be helpful for Maestro to either throw an error or a warning message when batch keys are found in the `run` block.

**Comment by FrankD412** [comment]
This one will actually be closed by #243

**Comment by FrankD412** [comment]
Closed by #243
